css-four-column-layout/index.html

```diff
@@ -18,58 +18,58 @@ <h1>How It Works</h1>
       <div class="row-body">
 
         <div class="column-one-fourth">
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <img class="img-screen" src="images/screen-1.png">
           </div>
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <img class="img-icon" src="images/switch-icon.png">
           </div>
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <h2>App Switching</h2>
           </div>
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <p>Toggles all your apps on and off.</p>
           </div>
         </div>
         <div class="column-one-fourth">
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <img class="img-screen" src="images/screen-2.png">
           </div>
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <img class="img-icon" src="images/filter-icon.png">
           </div>
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <h2>Trip Filtering</h2>
           </div>
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <p>Evaluates your trip requests.</p>
           </div>
         </div>
         <div class="column-one-fourth">
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <img class="img-screen" src="images/screen-3.png">
           </div>
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <img class="img-icon" src="images/slider-icon.png">
           </div>
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <h2>Service Filters</h2>
           </div>
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <p>Accepts only your ideal trips.</p>
           </div>
         </div>
         <div class="column-one-fourth">
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <img class="img-screen" src="images/screen-4.png">
           </div>
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <img class="img-icon" src="images/bars-icon.png">
           </div>
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
             <h2>Stats + Mileage</h2>
           </div>
-          <div class="row-one-fourth">
+          <div class="row-one-fourth text-center">
```

Comment:
You should remove `row-one-fourth` now that there is no styling correlated with it